### ISSUE TITLE
padduw not implemented on MIPS

### ISSUE THREAD

**Issue by GovanifY** [aquynh/capstone#940]
```
govanify@ThinkPadW500 ~ $ cat test.py 
from capstone import *

CODE= b"\x28\x0c\x00\x70"
md = Cs(CS_ARCH_MIPS, CS_MODE_32)
for i in md.disasm(CODE, 0x1000):
    print("0x%x:\t%s\t%s" %(i.address, i.mnemonic, i.op_str))
govanify@ThinkPadW500 ~ $ python test.py 
govanify@ThinkPadW500 ~ $ 
```

280c0070 should be disassembled as "padduw at, zero, zero" but is instead not implemented by capstone

**Comment by aquynh** [comment]
i am already planning to get a major update to Mips & other archs.

can you list all the missing instructions you can find here, for reference?

**Comment by GovanifY** [comment]
sure will check what I can for tomorrow, can't say it'll be a full list but I'll see what I can find missing

**Comment by wargio** [comment]
This cannot be patched because LLVM does not support MIPS EE SIMD instruction set.

Reference: https://psi-rockin.github.io/ps2tek/ ([archived](https://web.archive.org/web/20240624203830/https://psi-rockin.github.io/ps2tek/))

**Comment by Rot127** [comment]
For anyone bumping into this issue. There is the possibility to write the [table gen](https://llvm.org/docs/TableGen/ProgRef.html) file for the extension in LLVM. It will like not be up-streamed to LLVM because the extension is no longer in use. But we can use it to generate our files and support it in Capstone.
This is how we do it for the [PPC PS extension](https://github.com/capstone-engine/llvm-capstone/blob/auto-sync/llvm/lib/Target/PowerPC/PPCInstrPairedSingle.td).

**Comment by wargio** [comment]
The EE SIMD can be just a rizin plugin like we do for PPC VLE

**Comment by alinafoster4583** [comment]
Yes, there’s some kind of givеаway going on. I got some Ethereum there yesterday for different wallets. It’s unlikely they’ll take it back from me, but it’s a pity I didn’t have more wallets and couldn’t get more. But maybe it’s for the best.

<img width="373" alt="Image" src="https://github.com/user-attachments/assets/fea4b68b-a018-4d11-8158-5175a65e010b" />

<img width="369" alt="Image" src="https://github.com/user-attachments/assets/84f57ae7-7a09-4005-be87-ea6ae4c19e76" />
